doc/manpages/dinit-service.5.m4

```diff
@@ -360,6 +360,23 @@ This setting has no effect if the service is set to run on the console (via the
 \fBstarts\-on\-console\fR, or \fBshares\-console\fR options).
 The value is subject to variable substitution (see \fBVARIABLE SUBSTITUTION\fR).
 .TP
+\fBlogfile\-permissions\fR = \fIoctal-permissions-mask\fR
+Gives the permissions for the logfile specified using \fBlogfile\fR. Normally this will be 600 (user access
+only), 660 (user and group access), or 666 (all users).
+The default is 600.
+If logfile exist, thats perms will be ignored and the logfile permissions will be replaced with default or this option value.
```

Comment:
The grammar on this line needs a little work and "perms" is an abbreviation, how about:

"If the logfile already exists when the service starts, its permissions will be changed to those specified with this setting."